### DISCUSSION TITLE
Order by optional field

### DISCUSSION THREAD

**Discussion by divan** [geldata/gel#3791] [General]
How do I specify the order priority of NULL (missing) records for optional types?

I.e. if I have optional int field with values 1,2,3,(missing),4,5, the sort `ORDER BY .field` returns:

(missing),1,2,3,4,5

The question is how to tell that I want NULL (missing) values to be at the end, i.e.

1,2,3,4,5,(missing)

**Comment by elprans** [comment]
`order by .field empty last`

**Comment by divan** [comment]
Fantastic, thank you!
